lib/rules/order-in-components.js

```diff
@@ -259,6 +259,7 @@ module.exports = {
     }
 
     /**
+     * @param {RuleFixer} fixer
```

Comment:
This is not needed, It just fixes the tsc error on the main branch. This was implicitly any before.